_resourcetalk/style-guides-and-pattern-libraries-a-foundation-to-consistency.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Not sure if it's a formatting error somewhere but the link for this talk redirects back to the 'talks' page.

cc/ @bradfrost @maban